README.md

```diff
@@ -1,35 +1,35 @@
-# Support & Care for Maven
-This repository is used to track the work of [Support & Care for Maven](https://open-elements.com/support-care-maven/) as open and transparent as possible.
+# Support & Care for Apache Maven
```

Comment:
you will need to add few `&trade;`
see https://github.com/OpenElements/website/pull/89